### DISCUSSION TITLE
Vercel & Plastic SCM

### DISCUSSION THREAD

**Discussion by blackfire-std** [vercel/community#513] [Help] [answered]
Hi there !

At my company, we are using **Plastic SCM** and **Vercel** and would like to know if we could combine those to automate deployment and how.
Of course we could switch to **Github** but we would like to stick to our current **Plastic SCM** solution. Also it seems that **Plastic SCM** can synch with **Git** somehow : https://www.plasticscm.com/gitsync

Any suggestion is welcome :)
Thanks !

**Comment by swarnava** [comment, marked as answer]
Hi,

We don't have much expertise with Plastic SCM but I can imagine it should work like any other git workflow. If the platform support Git sync, you should be able to link the same Git repository with Vercel and any new commit to the Repository will automatically deploy changes to Vercel.

https://vercel.com/docs/concepts/deployments/overview#making-deployments

**Reply by blackfire-std** [reply]
Thank you very much for your fast answer.
Yeah this is something possible but we would like to avoid any git usage. What I mean is that we would like to avoid having a git just to be used as a intermediate.

**Reply by swarnava** [reply]
Yeah, I am assuming you don't need to touch to Git workflow at all. Any new changes on the SCM platform will automatically push change to Git and eventually on Vercel (whenever new commit receives). So far I can tell it from Vercel standpoint, but it would be interesting to know how they push the changes to Git. Please reach out to their support team for more info on this.

**Reply by blackfire-std** [reply]
Sure, thank you again.
